client/src/components/KeyboardShortcutDialog.vue

```diff
@@ -13,25 +13,26 @@ export default defineComponent({
     shortcuts: [
       ['Next frame', [['→']]],
       ['Previous frame', [['←']]],
-      [
-        'Change slices',
-        [
-          ['q', 'w'],
-          ['a', 's'],
-          ['z', 'x'],
-        ],
-      ],
+      ['Next frame', [['↓']]],
+      ['Previous frame', [['↑']]],
+      // [
+      //   'Change slices',
+      //   [
+      //     ['q', 'w'],
+      //     ['a', 's'],
+      //     ['z', 'x'],
+      //   ],
+      // ],
```

Comment:
Why are keyboard shortcuts being changed in this PR?

Reply:
My mistake, I didn't realize I left this commit on this branch. I undid these changes in [5df730a](https://github.com/OpenImaging/miqa/pull/355/commits/5df730aecb66c85b344c2346d14007fa8180b665)